### ISSUE TITLE
Collection of working keys

### ISSUE THREAD

**Issue by Gnagha** [Xyl2k/TSA-Travel-Sentry-master-keys#18]
Can someone reup or relink all working keys again in the right dimension? I'm a bit confused by all these files. A link would be very helpful because I want to print the keys with a metacrylate photopolymer gum and don't have a CAD programm right now. 

I want to try a better printer and better material, maybe then the keys work on more locks.

Thanks, G.

**Comment by MS3FGX** [comment]
The files in the repository are already the best working ones, as far as anyone knows. Most have not been tested, however, so there's no guarantee. 

Only 002, 003, and 007 are known to work (and even then, it seems to depend on lock type/manufacturer).

**Comment by Gnagha** [comment]
Okay .. then I'm going to print those in a better quality. I let you know if they work on different locks.

**Comment by MS3FGX** [comment]
What locks do you have access to? Any testing that can be done on the 4 remaining keys would be very helpful.

**Comment by Gnagha** [comment]
The shops I went to only have 007 and 002 :/

**Comment by johnnyxmas** [comment]
I'm at DerbyCon with IronGeek. We've got all 7 locks here. Will sit down and test sometime this weekend.

**Comment by Gnagha** [comment]
How did it work?

**Comment by johnnyxmas** [comment]
I was able to print working keys for 2,3,5 & 7. I also obtained measurements for 004, and believe I have a good file now.  The files I used are in my repo; I don't want to submit a pull request to merge them here until more people test. Please feel free to do so:

https://github.com/johnnyxmas/TSA-Travel-Sentry-master-keys

**Comment by amckenna** [comment]
I have access to 1, 2, 4, and 7 locks. I used the files from this repo and used shapeways.com to print them (figured it would be higher quality then borrowing someone extrusion printer).

001 - key fit but didn't turn
002 - key fit but didn't turn
004 - key did not fit
007 - key fit and it turned

I've included a picture of key 4 next to the lock.

![image](https://cloud.githubusercontent.com/assets/1251440/10254444/8cdd576e-68f8-11e5-9276-e0b714494247.png)

**Comment by johnnyxmas** [comment]
See my comment above.

**Comment by amckenna** [comment]
Yup I saw it, I'm just confirming that the ones in this repo are not good except 7.

**Comment by MS3FGX** [comment]
Well, not good for _your_ locks. You are the first to report problems for 002 since the new version, as far as I'm aware.

**Comment by elyajizi** [comment]
Hello , coming back here since it has been a life saving for me , got 2 big cases bought from Japan with 002 (TSA E150 and 734)  . Lost all keys and had only a dirty reproduction that worked hardly made by locksmith . I just last week found out that thisproject exist , and used your 092 file sent to a Chinese company that printed it on stainless steel  , working like a charms on 3 différents cases . Some polishing will be required since the print finish in stainless steel make key a bit hard to push on the lock , but definitely work 100%
Cheers
